### ISSUE TITLE
How to clear and rebuild graph?

### ISSUE THREAD

**Issue by caracal7** [nicoespeon/gitgraph.js#190]

**Comment by caracal7** [comment]
I found hacky solution how to reset graph and rebuild it with new data)
```
function ResetGraph() {
    gitgraph.HEAD = null;
    gitgraph.columnMax = 0;
    gitgraph.commitOffsetX = 0;
    gitgraph.commitOffsetY = 0;
    gitgraph.offsetX = 0;
    gitgraph.offsetY = 0;
    gitgraph.commits = [];
    gitgraph.branches = [];
}
MyRebuildCommits(data); // First build of graph
ResetGraph();         // Clear
MyRebuildCommits(newData); // Rebuild again
```

**Comment by fabien0102** [comment]
Nice hack 👍 I can add a `gitgraph.reset()` to the v2 (mmm I can't use the keyword `reset`… it's a git keyword ^^ Any naming idea ?)

**Comment by caracal7** [comment]
I think “clear()” is good solution)
But I don’t know that is inside gitgraph and maybe somewhere

**Comment by fabien0102** [comment]
https://github.com/nicoespeon/gitgraph.js/commit/3f59810fd250334eb87997411402630b605a972c

Included in the v2.0.0 (of course, for now the v2.0.0 don't draw graph yet, but the core is almost finish :wink:) @caracal7 thank again for the idea 😃
